Formula/git-flow.rb

```diff
@@ -42,13 +42,18 @@ class GitFlow < Formula
   def install
     system "make", "prefix=#{libexec}", "install"
     bin.write_exec_script libexec/"bin/git-flow"
-
     resource("completion").stage do
       bash_completion.install "git-flow-completion.bash"
       zsh_completion.install "git-flow-completion.zsh"
```

Comment:
We can also do:
```suggestion
      zsh_completion.install "git-flow-completion.zsh" => "_git_flow"
```

But all of this seems like something that should be fixed upstream

Reply:
That wouldn't work as far as I tested.
My suspicion is because it is installing from the bottle, that step is ignored.
Also it would break backwards compatibility. 

Fixing this upstream would break it for existing users. 
Also the hassle of fixing it upstream means touching a repo that hasn't been updated for seven years and then fixing new bottles, and so on.
This is quick fix that will work for both existing users sourcing it, as well as new installations.

Reply:
>  My suspicion is because it is installing from the bottle, that step is ignored.

If you're installing from a bottle locally it'll never pick up local modifications because it'll use the bottle definition of the formula.

> Fixing this upstream would break it for existing users.

It would stop being non-standard and work as ZSH users expect it to, yes

> Also the hassle of fixing it upstream means touching a repo that hasn't been updated for seven years and then fixing new bottles, tags, and so on.

Making new bottles is not that much work.

> This is quick fix that will work for both existing users sourcing it, as well as new installations.

But it will force homebrew to ship a workaround for eternity, [which we don't want](https://docs.brew.sh/Formula-Cookbook#patches)

Reply:
Ok, added the changes.